README.md

```diff
@@ -124,6 +139,59 @@ Example:
 }
 ```
 
+### Fetch content by delivery key (via Content Delivery Service 2)
+
+**Note:** In order to get content by it's delivery key via `getContentItemByKey()`, you must supply the `hubName` option to the client, you [find your Hub name under the settings section in Dynamic Content](https://docs.amplience.net/development/contentdelivery/readme.html#hubname).
```

Comment:
only if you have CD2 enabled

Reply:
Again we don't generally talk about provisioning of features in our SDK, I can add it if you think it will help make it clear.

Reply:
No problem - happy to go with the current pattern.